### DISCUSSION TITLE
defining center of sphere

### DISCUSSION THREAD

**Discussion by enghorbani** [neperfepx/neper#681] [Q&A] [answered]
I want to generate sphere particle in Neper with considering (0,0,0) as a center of sphere. How can I define? could you please help me?

I used this command, but it did not work: neper -T -n 100 -domain "sphere {r=0.076 c=(0,0,0)}" -o Particle_N1

**Comment by rquey** [comment, marked as answer]
This is not something Neper can do, and not Neper syntax...

Duplicates #682.

**Reply by enghorbani** [reply]
Thank you so much
